deploy/manifests/alpha/basic.yml

```diff
@@ -0,0 +1,15 @@
+---
+applications:
+- name: alpha-basic
+  memory: 1024M
+  instances: 2
+  buildpack: java_buildpack
+  health-check-http-endpoint: /records
+  random-route: true
+  services:
+    - alpha-db
```

Comment:
The connection strings are ephemeral and can change on bind/unbind/probably other things. We can't hardcode the them into the `config.yaml`, [so we have to do read them from the environment at runtime](https://github.com/openregister/openregister-java/blob/master/src/main/java/uk/gov/register/configuration/DatabaseManager.java#L22). We do however have to [provide a dummy value so Dropwizard doesn't barf](https://github.com/openregister/deployment/blob/master/ansible/roles/openregister_config/templates/openregister-config.yaml.j2#L17).

You're also right that we're describing that this app is even able to connect to this database—I believe there's some Postgres configuration that expects to only see connections from certain IPs.